README.md

```diff
@@ -82,12 +82,12 @@ Queries can be used to fetch data stored/managed by the database. For example, w
   1. [Querying the configuration](./docs/curl/query.md#querying-the-cluster-configuration)
   2. [Querying a user information/credentials/privileges](./docs/curl/query.md#querying-the-user-information)
   3. [Checking the existance of a database](./docs/curl/query.md#checking-the-database-existance)
-  4. [Querying data]()
-  5. [Querying provenance of data or user]()
-  6. [Querying a block](./docs/curl/query.md#querying-a-block-header)
+  4. [Querying data](./docs/curl/datatx.md#checking-the-existence-of-the-state)
+  5. [Querying provenance of data or user](./docs/curl/provenance.md)
+  6. Querying a block - [here](./docs/curl/query.md#querying-a-block-header) or [here](./docs/curl/ledger.md#block-header-query)
   7. [Querying a transaction]()
-  8. [Querying proof of existance of a transaction]
-  9. [Querying proof of existance of data]
+  8. [Querying proof of existence of a transaction](./docs/curl/ledger.md#transaction-proof-query)
+  9. [Querying proof of existance of data](./docs/curl/ledger.md#state-proof-query)
```

Comment:
I think we should simplify the structure here by using the `provenance` as one category, and `ledger` as a second, just as we have on our API. under each category we can detail the type of info we provide.